### PR TITLE
chore(cleanup): removes APIs deprecated in 1.9

### PR DESCRIPTION
**Pull request by mrclay**
BREAKING CHANGES:
See the additions to `docs/guides/upgrading.rst` made in this commit.

### CODE REVIEW

**Comment by mrclay** [comment]
Therapeutic.

**Comment by juho-jaakkola** [comment]
:)

**Comment by juho-jaakkola** [comment]
LGTM

File: docs/guides/upgrading.rst [hunk 1]
```diff
@@ -20,17 +20,105 @@ Removed views
 Removed functions/methods
 -------------------------
 
- * ``ElggFile::setFilestore``: ElggFile objects can no longer use custom filestores.
- * ``ElggFile::size``: Use ``getSize``
+All the functions in ``engine/lib/deprecated-1.9.php`` were removed. See https://github.com/Elgg/Elgg/blob/2.x/engine/lib/deprecated-1.9.php for these functions. Each ``@deprecated`` declaration includes instructions on what to use instead.
```
Comment (hypeJunction) [inline docs/guides/upgrading.rst]: We might get rid of 2.x branch eventually, so perhaps a tag should be used?

**Comment by hypeJunction** [comment]
Seems fine

**Pull request merged**